Gulpfile.js

```diff
@@ -79,29 +98,57 @@ gulp.task('js', () =>
         'assets/js/topic-suggest.js',
         'assets/js/tribune-pick.js',
         'assets/js/zen-mode.js',
-    ], { base: '.' })
-        .pipe(sourcemaps.init({ loadMaps: true }))
-        .pipe(concat('script.js', { newline: ';\r\n' }))
```

Comment:
T'as besoin du `concat` dans tous les cas et ça bouffe rien comme temps

Reply:
"This will concat files by your operating systems newLine. It will take the base directory from the first file that passes through it." source: https://www.npmjs.com/package/gulp-concat

Je viens de vérifier 2ms... x)